src/controllers/team.controller.ts

```diff
@@ -59,3 +60,37 @@ export async function updateMembers(
       );
   }
 }
+
+export async function addMemberByName(
+  req:RequestWithUser,
+  res:Response,
+): Promise<Response | void> {
+
+  const id: string = req.params.id;
+  const userToAdd: string = req.params.username;
+
+  const user = await userService.getByUsername(userToAdd);
+  const team = await teamService.getById(id);
+
+  if (!user) {
```

Comment:
Also we have implemented checks of existence on repository level, so if there are no entities with such ids, exception will be thrown from repository.

So catch this exception in this method and return their code and their message, not your custom one